core/src/components/item-sliding/item-sliding.tsx

```diff
@@ -252,7 +253,23 @@ export class ItemSliding implements ComponentInterface {
     return !!(this.rightOptions || this.leftOptions);
   }
 
+  private toggleContentScrollY() {
+    const content = this.el.closest('ion-content');
```

Comment:
Can we grab the content in `connectedCallback` so that we do not need to query for it every time the gesture starts?